pkg/handlers/generic/lifecycle/ccm/nutanix/handler.go

```diff
@@ -148,6 +149,7 @@ func (p *provider) Apply(
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: cluster.Namespace,
 			Name:      "nutanix-ccm-" + cluster.Name,
+			Labels:    utils.NewLabels(utils.WithMove()),
```

Comment:
Do these not get moved with `clusterctl move` like ClusterResourceSets do?

Reply:
actually, they should be moved if they are owned by the cluster object. nice catch removing